gremlin/values.yaml

```diff
@@ -253,3 +253,20 @@ ssl:
   # sets the SSL_CERT_DIR environment variable on the both agents. Unlike ssl.certFile, this value accepts only a
   # path to an existing directory on the Kubernetes nodes. This value is ignored when blank or absent.
   certDir:
+
+containerDrivers:
```

Comment:
The use of values here allow customers to override these if they so choose.

While that isn't necessary it remains a value use case for power-users

Reply:
This is great! One problem here though is that `gremlin` does no know to look anywhere but the paths listed here. If some cluster puts their Docker socket at `/opt/custom/path/docker.sock`, `gremlin` still needs it to be mounted at `/var/run/docker.sock` inside the container.

Perhaps based on the defined `name`, we can fix the `mountPath` value to something Gremlin expects?